src/main/java/it/pagopa/pn/mandate/services/mandate/v1/MandatePrivateService.java

```diff
@@ -28,17 +28,23 @@ public MandatePrivateService(MandateDao mandateDao, MandateEntityInternalMandate
 
     public Flux<InternalMandateDto> listMandatesByDelegate(String internaluserId, String mandateId) {
         // nelle invocazioni tra servizi mi interessano SEMPRE solo le deleghe ATTIVE
-        if (log.isInfoEnabled())
-                    log.info("listing private mandates by delegate for " + internaluserId);
+        log.info("listing private mandates by delegate for internaluserId={} mandateId={}", internaluserId, mandateId);
         return mandateDao.listMandatesByDelegate(internaluserId, StatusEnumMapper.intValfromStatus(StatusEnum.ACTIVE), mandateId)
-                .map(mandateEntityInternalMandateDtoMapper::toDto);
+                .map(mandateEntityInternalMandateDtoMapper::toDto)
+                .map(mand -> {
```

Comment:
Sarebbe più elegante utilizzare un onNext o ancora meglio onSuccess in questo caso, per loggare (tutto in-line), invece di utlizzare map.